### DISCUSSION TITLE
Atomize `Mutect2` MNV variants?

### DISCUSSION THREAD

**Discussion by sorelfitzgibbon** [uclahs-cds/pipeline-call-sSNV#172] [Q&A] [answered]
MNVs are in-phase neighboring SNVs.  On somewhat rare occasions they can code for a codon change which is not reflected in any of the individual SNVs, e.g. a stop codon. 

But this is an SNV pipeline so the SNVs have been separated from MNVs and indels.  I'm not sure whether to break the MNVs into their component SNVs and merge them back in with the other SNVs.   This seems like the right thing to do, along with keeping the MNVs VCF in the output, but it could be confusing.

Variant Counts
```
..._pass_indels.vcf.gz 838
..._pass_mnps.vcf.gz 88
..._pass_snps.vcf.gz 1691
..._pass.vcf.gz 2617
```

**Comment by pboutros** [comment, marked as answer]
Definitely no to splitting MNVs.  It’s fundamentally different from a cancer perspective to have:
AA --> AB --> BB (two mutational events)
Vs.
AA --> BB (one mutational event)

We have to trust that the caller is getting this correct and is suggesting that an MNV is one event not two (or at least that there’s no evidence to support two)


From: Sorel Fitz-Gibbon ***@***.***>
Sent: Wednesday, April 12, 2023 6:52 PM
To: uclahs-cds/pipeline-call-sSNV ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [uclahs-cds/pipeline-call-sSNV] Atomize `Mutect2` MNV variants? (Discussion #172)


CAUTION - EXTERNAL EMAIL:Do not click links or open attachments unless you recognize the sender.

MNVs are in-phase neighboring SNVs. On somewhat rare occasions they can code for a codon change which is not reflected in any of the individual SNVs, e.g. a stop codon.

But this is an SNV pipeline so the SNVs have been separated from MNVs and indels. I'm not sure whether to break the MNVs into their component SNVs and merge them back in with the other SNVs. This seems like the right thing to do, along with keeping the MNVs VCF in the output, but it could be confusing.

Variant Counts

..._pass_indels.vcf.gz 838

..._pass_mnps.vcf.gz 88

..._pass_snps.vcf.gz 1691

..._pass.vcf.gz 2617

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/uclahs-cds/pipeline-call-sSNV/discussions/172__;!!F9wkZZsI-LA!D9NDKCDOxFV07WfZsdpN_80ao_q8cF7kk6M-FSX2UNTMWUGoYUZ8QZXb3Y7ZJCBkVoaoAv2wOpjUJ1Oe_NkV7VbCCxzUa84$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AB3YOXT3XJZ3C3XUTOU43TDXA5L33ANCNFSM6AAAAAAW4NQE74__;!!F9wkZZsI-LA!D9NDKCDOxFV07WfZsdpN_80ao_q8cF7kk6M-FSX2UNTMWUGoYUZ8QZXb3Y7ZJCBkVoaoAv2wOpjUJ1Oe_NkV7VbClkgHmt8$>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.******@***.***>>

________________________________

UCLA HEALTH SCIENCES IMPORTANT WARNING: This email (and any attachments) is only intended for the use of the person or entity to which it is addressed, and may contain information that is privileged and confidential. You, the recipient, are obligated to maintain it in a safe, secure and confidential manner. Unauthorized redisclosure or failure to maintain confidentiality may subject you to federal and state penalties. If you are not the intended recipient, please immediately notify us by return email, and delete this message from your computer.

**Comment by sorelfitzgibbon** [comment]
OK, but if I understand correctly we won't be using these MNVs downstream (in this pipeline).  Would they have been called as SNVs if Mutect2 was not looking for MNVs?   You would rather they are not counted at all?

**Comment by pboutros** [comment]
I’m not sure of the question here?

**Reply by sorelfitzgibbon** [reply]
Are you saying to keep the MNVs in the `mutect2.snvs.vcf.gz`  that will be intersected with the SNVs from the other tools?   If so, 
 - what about the indels, keep or not?   All of the other tools output an SNV-only file (Strelka outputs an additional indel file)
 - the intersection will have to be done carefully to make sure overlaps are considered.

**Comment by pboutros** [comment]
Oh, I didn’t make any comment on that.  I’m simply saying that it’s not necessarily biologically correct to treat an MNV as its constitutive SNVs
